testing/opae-c/test_usrclk_c.cpp

```diff
@@ -102,25 +102,33 @@ class usrclk_c_p : public ::testing::TestWithParam<std::string> {
  *             and the fn returns FPGA_OK.<br>
  */
 TEST_P(usrclk_c_p, get) {
-  uint64_t low = 0;
-  uint64_t high = 0;
+  uint64_t low = 999;
+  uint64_t high = 999;
   EXPECT_EQ(fpgaGetUserClock(accel_, &high, &low, 0), FPGA_OK);
-  EXPECT_NE(low, 0);
-  EXPECT_NE(high, 0);
+  EXPECT_NE(low, 999);
+  EXPECT_NE(high, 999);
 }
 
+// TODO: Fix user clock test for DCP
+INSTANTIATE_TEST_CASE_P(usrclk_c, usrclk_c_p,
+                        ::testing::ValuesIn(test_platform::platforms({"skx-p"})));
+
+class usrclk_c_hw_p : public usrclk_c_p{
+  protected:
+    usrclk_c_hw_p() {};
+};
 /**
  * @test       set
  * @brief      Test: fpgaSetUserClock
  * @details    When fpgaSetUserClock is called with valid parameters,<br>
  *             the fn returns FPGA_OK.<br>
  */
-TEST_P(usrclk_c_p, DISABLED_set) {
+TEST_P(usrclk_c_hw_p, set) {
   uint64_t low = 25;
-  uint64_t high = 1200;
+  uint64_t high = 600;
```

Comment:
These values are acceptable for all current hardware platforms?

Reply:
Yeah, @anandaravuri said the range 25-600 will return FPGA_OK.

Reply:
Awesome, thanks!